.gitignore

```diff
@@ -0,0 +1,4 @@
+application.css
+.sass-cache
+application.css.map
```

Comment:
You are gonna have multiple `*.css` and `*.css.map`. Add those rules instead of one for each file